platforms/corda-ent/charts/node/templates/deployment.yaml

```diff
@@ -209,8 +209,10 @@ spec:
           cp CM-FILES/*.sh bin/
           cp corda.jar bin/
           chmod +x bin/*
-          yes | keytool -importcert -file {{ .Values.workspacePath }}/certificates/tlscerts/networkmap.crt -storepass changeit -alias networkmap -keystore /usr/lib/jvm/zulu8.33.0.1-jdk8.0.192-linux_musl_x64/jre/lib/security/cacerts
-          yes | keytool -importcert -file {{ .Values.workspacePath }}/certificates/tlscerts/idman.crt -storepass changeit -alias idman -keystore /usr/lib/jvm/zulu8.33.0.1-jdk8.0.192-linux_musl_x64/jre/lib/security/cacerts
+          # to clean network-parameters on every restart
```

Comment:
Line 147, 152 needs to be changed
IDMAN_CERT=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["idman.crt"]')
As here the name of the idman.crt should be like {{ peer.name }}.crt

Reply:
No, why? Why should it be peer.name.crt? These are idman public certs and not peer certs

Reply:
Because, in vault the crts are being stored as peer.name.crt, if we name idman and nmap differently, it is not able to pick from vault

Reply:
![image](https://user-images.githubusercontent.com/67091468/92701925-34711600-f36e-11ea-924e-53f6ddbce9d1.png)

Reply:
Ya, I mean that it should be the actual idman name and networkmap name

Reply:
Yes, then we need to pass the names in the values.yaml (which I think we are already doing) and use those variables here